### PR TITLE
modify security overview

### PR DESCRIPTION
**Pull request by kbhawkey**
- Minor style cleanup
- Some rewording of sections (remove "master" term)
- Attempt to remove links to third party content.

### CODE REVIEW

File: content/en/docs/concepts/security/overview.md [hunk 1]
```diff
@@ -3,59 +3,55 @@ reviewers:
...
+for running workloads securely in their environment.
 
-### Cloud Provider Security Table
+### Cloud provider security
 
+If you are running a Kubernetes cluster on your own hardware or a different cloud provider,
+consult your documentation for security best practices.
+Here are links to some of the popular cloud providers' security documentation:
 
+{{< table caption="cloud provider security" >}}
```
Comment (sftim) [inline content/en/docs/concepts/security/overview.md:54]: nit:
```suggestion
{{< table caption="Cloud provider security" >}}
```
?

File: content/en/docs/concepts/security/overview.md [hunk 2]
```diff
@@ -114,51 +112,49 @@ Quality of Service (and Cluster resource management) | https://kubernetes.io/doc
...
+links to explore this topic:
 
 Area of Concern for Containers | Recommendation |
------------------------------- | ------------ |
-Container Vulnerability Scanning and OS Dependency Security | As part of an image build step or on a regular basis you should scan your containers for known vulnerabilities with a tool such as [CoreOS's Clair](https://github.com/coreos/clair/)
-Image Signing and Enforcement | Two other CNCF Projects (TUF and Notary) are useful tools for signing container images and maintaining a system of trust for the content of your containers. If you use Docker, it is built in to the Docker Engine as [Docker Content Trust](https://docs.docker.com/engine/security/trust/content_trust/). On the enforcement piece, [IBM's Portieris](https://github.com/IBM/portieris) project is a tool that runs as a Kubernetes Dynamic Admission Controller to ensure that images are properly signed via Notary before being admitted to the Cluster.
+------------------------------ | -------------- |
+Container Vulnerability Scanning and OS Dependency Security | As part of an image build, periodically scan your containers for known vulnerabilities.
```
Comment (sftim) [inline content/en/docs/concepts/security/overview.md:122]: You can scan at build time and you can periodically scan built containers. I don't think you can periodically do a scan as part of the build process.

Comment (kbhawkey) [reply content/en/docs/concepts/security/overview.md]: Thanks for the review!

File: content/en/docs/concepts/security/overview.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/concepts/security/overview.md]: I updated text.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-21493--kubernetes-io-master-staging.netlify.app/docs/concepts/security/overview/

**Pull request merged**